docs/dev_guide/documentation.rst

```diff
@@ -50,9 +50,11 @@ SunPy Specific Rules
 
 .. code-block:: python
```

Comment:
make this not a Python code block? This is rst right? (I assume that would also fix the warning?)

Reply:
Well it was meant to be a python docstring block.